library/node

```diff
@@ -1,34 +1,43 @@
 # maintainer: Joyent Image Team <image-team@joyent.com> (@joyent)
 
-0.10.36: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
-0.10: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
-0: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
-latest: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
+0.10.36: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10
+0.10: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10
 
 0.10.36-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
 0.10-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
-0-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
 
-0.10.36-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
-0.10-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
-0-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
+0.10.36-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10/slim
+0.10-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10/slim
 
-0.11.16: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11
-0.11: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11
+0.11.16: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11
+0.11: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11
 
 0.11.16-onbuild: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/onbuild
 0.11-onbuild: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/onbuild
 
-0.11.16-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/slim
-0.11-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/slim
+0.11.16-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11/slim
+0.11-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11/slim
```

Comment:
No, I don't believe so, but that's not why I keep 0.11.16 for the short term.

There was a long wait for 0.12 and I'm sure many got (reasonably) impatient and opted to use 0.11.* to access certain features. There are probably modules out there (like koa for example) that require 0.11.*. In theory, yeah, you should be able to switch to 0.12.0 but I want to give people time to make that switch. Just removing 0.11.16 outright feels a bit too abrupt to me.